sensors/nxt_i2c_sensor_defs.c

```diff
@@ -2582,6 +2610,149 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 			},
 		},
 	},
+	[MS_PPS58_NX] = {
+		/**
+		 * @vendor_name: mindsensors.com
+		 * @vendor_part_number: PPS58-Nx
+		 * @vendor_part_name: Digital Pneumatic Pressure Sensor
+		 * @vendor_website: http://mindsensors.com/index.php?module=pagemaster&PAGE_user_op=view_page&PAGE_id=150
```

Comment:
I almost forgot. mindsensors has updated their website since I did this. new link is

    http://www.mindsensors.com/ev3-and-nxt/127-digital-pneumatic-pressure-sensor-for-nxt-or-ev3